libs/concurrency/tests/unit/lockfree_fifo.cpp

```diff
@@ -59,29 +60,30 @@ void worker_thread(std::uint64_t num_thread)
 
 int main(int argc, char** argv)
 {
-    using hpx::program_options::variables_map;
-    using hpx::program_options::options_description;
-    using hpx::program_options::value;
-    using hpx::program_options::store;
     using hpx::program_options::command_line_parser;
     using hpx::program_options::notify;
+    using hpx::program_options::options_description;
+    using hpx::program_options::store;
+    using hpx::program_options::value;
+    using hpx::program_options::variables_map;
 
     variables_map vm;
 
-    options_description
-        desc_cmdline("Usage: " HPX_APPLICATION_STRING " [options]");
+    options_description desc_cmdline(
+        "Usage: " HPX_APPLICATION_STRING " [options]");
 
-    desc_cmdline.add_options()
-        ("help,h", "print out program usage (this message)")
-        ("threads,t", value<std::uint64_t>(&threads)->default_value(2),
-         "the number of worker threads inserting objects into the fifo")
-        ("items,i", value<std::uint64_t>(&items)->default_value(500000),
-         "the number of items to create per queue")
-    ;
+    desc_cmdline.add_options()(
+        "help,h", "print out program usage (this message)")("threads,t",
+        value<std::uint64_t>(&threads)->default_value(2),
+        "the number of worker threads inserting objects into the fifo")(
+        "items,i", value<std::uint64_t>(&items)->default_value(500000),
+        "the number of items to create per queue");
```

Comment:
I know it's annoying, but could we 'clang-unformat' the command line options, please?

Reply:
Yeah, no worries :) I left this as a draft exactly for these kind of things. I'll fix it.